src/scaler/manager.rs

```diff
@@ -596,13 +598,15 @@ where
                             Some(Box::new(ActorDaemonScaler::new(
                                 snapshot_data.clone(),
                                 props.image.to_owned(),
+                                actor_id,
                                 lattice_id.to_owned(),
                                 name.to_owned(),
                                 p.to_owned(),
                                 &component.name,
                             )) as BoxedScaler)
                         }
-                        (LINKDEF_TRAIT, TraitProperty::Linkdef(p)) => {
+                        (LINK_TRAIT, TraitProperty::Link(p)) => {
+                            // TODO: look for actor or provider components
```

Comment:
Not sure I understand the TODO here

Reply:
Bad description here, all I meant was that we need to have links on actors and providers both now